mantle/xilinx/mantle3/MUX.py

```diff
@@ -129,14 +130,14 @@ def MuxN(height, **kwargs):
     elif height == 16:
         return Mux16(**kwargs)
 
-def DefineMux(height=2, width=1):
+def DefineMux(height=2, width=1, T=None):
```

Comment:
why is this interface have `width` and `T`? Do we want to move to a unified interface with just `T`? Is this just for backwards compatibility?

Reply:
We do want to move to a unified interface, but this is just updating them to match the existing interface.  `T` was added after `width` (which used to be the only way to construct it), In the future I think we should use `T`, but we may be able to support `width` and gracefully deprecate it.  However, these changes just update the mantle3/6 implementations to be consistent with the ice40/coreir implementations, I think the interface changes should come in a separate change set.